saturn.opam.template

```diff
@@ -0,0 +1,3 @@
+pin-depends: [
+  [ "multicore-bench.dev" "git+https://github.com/ocaml-multicore/multicore-bench#bdbdc6787d670725349c880581be770638fbb05b" ]
```

Comment:
The multicore-bench package needs to be published and this pin-depend removed before merging.